build/docker/intel-fpga-admissionwebhook.Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM golang:1.11 as builder
-ARG DIR=/go/src/github.com/intel/intel-device-plugins-for-kubernetes
+FROM golang:1.12 as builder
```

Comment:
This changes in #199 so both fpga Dockerfile changes can be dropped.

Reply:
#199 doesn't update `ARG DIR=/go/src/github.com/intel/intel-device-plugins-for-kubernetes` which implicitly switches go mod off.

Reply:
please update this PR based on the merged changes from PR #199

Reply:
Ok. Resolved the conflict and force-pushed.